src/framework/preferences/js/SyllabificationEnactor.js

```diff
@@ -227,11 +227,15 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
         injectPromise.then(function () {
             hyphenator = fluid.getGlobalValue(globalPath);
             promise.resolve(hyphenator);
-        }, function () {
-            var errorMessage = fluid.stringTemplate(that.options.strings.patternLoadError, {src: src});
-            // Pass along arguments fired for the rejection, but with our errorMessage as the first argument
-            var args = [errorMessage].concat(fluid.values(arguments));
-            that.events.onError.fire.apply(null, args);
+        }, function (error) {
+            var errorInfo = {
+                src: src,
+                errorMsg: typeof(error) === "string" ? error : ""
```

Comment:
Still not quite right - now you throw away the "error" object completely when is it not a string

Reply:
Thanks, I had meant to add it to the fluid.log call as well.